### ISSUE TITLE
The predicted_image and inpainted_image after passing the input and mask image through the model are none in remove_anything

### ISSUE THREAD

**Issue by HG2407** [geekyutao/Inpaint-Anything#126]
The input image and mask image are both correct and have appropriate values but the predicted image and the inpainted image after passing the batch through the model in "lama_inpaint.py" are filled with 'nan'. Why is this happening. Below I am showing one of the outputs:

```
batch:{'image': tensor([[[[0.5294, 0.5294, 0.5255,  ..., 0.2667, 0.2157, 0.1961],
          [0.5294, 0.5333, 0.5294,  ..., 0.2824, 0.2000, 0.1804],
          [0.5294, 0.5333, 0.5255,  ..., 0.2549, 0.1961, 0.1725],
          ...,
          [0.1961, 0.1922, 0.1882,  ..., 0.4980, 0.4824, 0.4784],
          [0.2000, 0.1961, 0.2000,  ..., 0.5490, 0.5373, 0.5255],
          [0.2000, 0.1882, 0.1804,  ..., 0.5647, 0.5569, 0.5569]],

         [[0.4431, 0.4431, 0.4471,  ..., 0.2078, 0.1569, 0.1569],
          [0.4431, 0.4471, 0.4471,  ..., 0.2235, 0.1412, 0.1333],
          [0.4471, 0.4510, 0.4471,  ..., 0.2000, 0.1451, 0.1216],
          ...,
          [0.0980, 0.1059, 0.1020,  ..., 0.3725, 0.3608, 0.3647],
          [0.1020, 0.1098, 0.1098,  ..., 0.4275, 0.4235, 0.4118],
          [0.1020, 0.0980, 0.0980,  ..., 0.4431, 0.4353, 0.4392]],

         [[0.3529, 0.3569, 0.3569,  ..., 0.1961, 0.1529, 0.1373],
          [0.3529, 0.3608, 0.3608,  ..., 0.2039, 0.1294, 0.1216],
          [0.3569, 0.3608, 0.3608,  ..., 0.1765, 0.1255, 0.1059],
          ...,
          [0.0784, 0.0824, 0.0784,  ..., 0.2902, 0.2745, 0.2784],
          [0.0784, 0.0824, 0.0863,  ..., 0.3412, 0.3333, 0.3216],
          [0.0745, 0.0745, 0.0745,  ..., 0.3608, 0.3569, 0.3490]]]],
       device='cuda:0'), 'mask': tensor([[[[0, 0, 0,  ..., 0, 0, 0],
          [0, 0, 0,  ..., 0, 0, 0],
          [0, 0, 0,  ..., 0, 0, 0],
          ...,
          [0, 0, 0,  ..., 0, 0, 0],
          [0, 0, 0,  ..., 0, 0, 0],
          [0, 0, 0,  ..., 0, 0, 0]]]], device='cuda:0'), 'predicted_image': tensor([[[[nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          ...,
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan]],

         [[nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          ...,
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan]],

         [[nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          ...,
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan]]]], device='cuda:0'), 'inpainted': tensor([[[[nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          ...,
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan]],

         [[nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          ...,
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan]],

         [[nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          ...,
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan],
          [nan, nan, nan,  ..., nan, nan, nan]]]], device='cuda:0'), 'mask_for_losses': tensor([[[[0, 0, 0,  ..., 0, 0, 0],
          [0, 0, 0,  ..., 0, 0, 0],
          [0, 0, 0,  ..., 0, 0, 0],
          ...,
          [0, 0, 0,  ..., 0, 0, 0],
          [0, 0, 0,  ..., 0, 0, 0],
          [0, 0, 0,  ..., 0, 0, 0]]]], device='cuda:0')}
      
      
```

it arises in this particular function:
````
@torch.no_grad()
def inpaint_img_with_lama(
        img: np.ndarray,
        mask: np.ndarray,
        config_p: str,
        ckpt_p: str,
        mod=8,
        device="cuda"
):
    assert len(mask.shape) == 2
    if np.max(mask) == 1:
        mask = mask * 255
    img = torch.from_numpy(img).float().div(255.)
    mask = torch.from_numpy(mask).float()
    predict_config = OmegaConf.load(config_p)
    predict_config.model.path = ckpt_p
    # device = torch.device(predict_config.device)
    device = torch.device(device)

    train_config_path = os.path.join(
        predict_config.model.path, 'config.yaml')

    with open(train_config_path, 'r') as f:
        train_config = OmegaConf.create(yaml.safe_load(f))

    train_config.training_model.predict_only = True
    train_config.visualizer.kind = 'noop'

    checkpoint_path = os.path.join(
        predict_config.model.path, 'models',
        predict_config.model.checkpoint
    )
    model = load_checkpoint(
        train_config, checkpoint_path, strict=False, map_location='cpu')
    
    model.freeze()
    if not predict_config.get('refine', False):
        model.to(device)

    batch = {}
    batch['image'] = img.permute(2, 0, 1).unsqueeze(0)
    batch['mask'] = mask[None, None]
    unpad_to_size = [batch['image'].shape[2], batch['image'].shape[3]]
    batch['image'] = pad_tensor_to_modulo(batch['image'], mod)
    batch['mask'] = pad_tensor_to_modulo(batch['mask'], mod)
    batch = move_to_device(batch, device)
    batch['mask'] = (batch['mask'] > 0) * 1
    print(f"batch before inpainting:{batch}")
    batch = model(batch)
    print(f"batch:{batch}")
    cur_res = batch[predict_config.out_key][0].permute(1, 2, 0)
    cur_res = cur_res.detach().cpu().numpy()
    print(f"cur_res:{cur_res}")

    if unpad_to_size is not None:
        orig_height, orig_width = unpad_to_size
        cur_res = cur_res[:orig_height, :orig_width]

    cur_res = np.clip(cur_res * 255, 0, 255).astype('uint8')
    return cur_res
```
    
    
as you can see the input image is normalized within the valid range of 0 to 1 and mask image is binary. So why are the predicted and inpainted image coming out as "nan" ?
